pcsx2/gui/Dialogs/AboutBoxDialog.cpp

```diff
@@ -54,62 +54,47 @@ Dialogs::AboutBoxDialog::AboutBoxDialog( wxWindow* parent )
 		L"%s: CKemu, Falcon4ever",
 		_("Previous versions"), _("Betatesting"), _("Webmasters"));
 
-
-	wxString LabelGreets = wxsFormat( 
+	wxString contribsString = wxsFormat(
 		L"Hiryu and Sjeep (libcdvd / iso filesystem), nneeve (fpu and vu), n1ckname (compilation guides), Shadow Lady"
 		L"\n\n"
-			L"%s: ChickenLiver (Lilypad), Efp (efp), "
+		L"%s: ChickenLiver (Lilypad), Efp (efp), "
 		L"Gabest (Gsdx, Cdvdolio, Xpad),  Zeydlitz (ZZogl)"
 		L"\n\n"
-			L"%s: black_wd, Belmont, BGome, _Demo_, Dreamtime, "
-			L"F|RES, Jake.Stine, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos",
-			_("Plugin Specialists"), _("Special thanks to"));
-
-	// This sizer holds text of the authors and a logo!
-	wxFlexGridSizer& AuthLogoSizer = *new wxFlexGridSizer( 2, 0, StdPadding );
-	AuthLogoSizer.AddGrowableCol(0, 4);
-	AuthLogoSizer.AddGrowableCol(1, 3);
+		L"%s: black_wd, Belmont, BGome, _Demo_, Dreamtime, "
+		L"F|RES, Jake.Stine, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos",
+		_("Plugin Specialists"), _("Special thanks to"));
 
-	// this sizer holds text of the contributors/testers, and a ps2 image!
-	wxBoxSizer& ContribSizer = *new wxBoxSizer( wxHORIZONTAL );
+	wxFlexGridSizer& boxesContainer = *new wxFlexGridSizer(2, 0, StdPadding);
+	boxesContainer.AddGrowableCol(0, 1);
+	boxesContainer.AddGrowableCol(1, 1);
 
-	wxStaticBoxSizer& aboutUs	= *new wxStaticBoxSizer( wxVERTICAL, this );
-	wxStaticBoxSizer& contribs	= *new wxStaticBoxSizer( wxVERTICAL, this );
+	wxStaticBoxSizer& developsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
+	wxStaticBoxSizer& contribsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
 
-	pxStaticText& label_auth	= Text( LabelAuthors ).SetMinWidth(240);
-	pxStaticText& label_greets	= Text( LabelGreets ).SetMinWidth(200);
+	pxStaticText& developsText = Text(developsString).SetMinWidth(220);
+	pxStaticText& contribsText = Text(contribsString).SetMinWidth(220);
```

Comment:
Possibly set both to 240? It makes the Windows/Linux about boxes look more similar to each other.